src/core/analysis/mcmc/MarginalLikelihoodEstimator.cpp

```diff
@@ -111,7 +120,11 @@ MarginalLikelihoodEstimator::MarginalLikelihoodEstimator(const path &fn, const s
         
             // get the likelihood entry
             tmp = columns[likelihoodColumnIndex];
-            double l = atof( tmp.c_str() );
+            try {
+                l = std::stod(tmp.c_str());
```

Comment:
Doesn't std::stod takes a std::string as input instead of a c-array of characters?

Reply:
you're right, I just copied the previous code - fixed now